src/tools/rbd/action/MirrorPool.cc

```diff
@@ -259,16 +259,48 @@ void get_enable_arguments(po::options_description *positional,
 }
 
 int execute_enable_disable(const std::string &pool_name,
-                           rbd_mirror_mode_t mirror_mode) {
+                           const std::string &mode) {
   librados::Rados rados;
   librados::IoCtx io_ctx;
+  rbd_mirror_mode_t next_mirror_mode;
+  rbd_mirror_mode_t current_mirror_mode;
+
+  if (mode == "disable") {
```

Comment:
Minor: perhaps just pass in both the string and the enum so that you don't need to move this logic here (and so that "rbd mirror pool enable disable" isn't a valid selection).